backend/document-service/src/main/kotlin/com/nosql/document/controller/DocumentController.kt

```diff
@@ -72,6 +73,16 @@ class DocumentController(
         @Valid @RequestBody documentDto: DocumentDto,
     ) = documentService.save(userAuthInfo.userId, documentDto)
 
+    @PutMapping(
+        value = [ID_URL_PATH],
+        consumes = [APPLICATION_JSON_VALUE],
+        produces = [APPLICATION_JSON_VALUE],
+    )
```

Comment:
I think this is `PATCH` - We are updating not whole object, but only few fields

Reply:
fixed